content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -44,15 +44,16 @@ If you are running a version of Kubernetes other than v{{< skew currentVersion >
 check the documentation for that version.
 {{< /note >}}
 
-
 <!-- body -->
 ## Install and configure prerequisites
 
-The following steps apply common settings for Kubernetes nodes on Linux. 
+The following steps apply common settings for Kubernetes nodes on Linux.
 
 You can skip a particular setting if you're certain you don't need it.
 
-For more information, see [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) or the documentation for your specific container runtime.
+For more information, see
+[Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements)
+or the documentation for your specific container runtime.
```

Comment:
What's the point to split one line in 3? I see this pattern everywhere in this PR and just wondering if there is any specific requirement or recommendation regarding this or it's just a matter of taste?

Reply:
Not sure but I think there's a preference to adhere to the [Style Guide: Line Length](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#line-length) recommendation.

 All l know is that I've been directed to shorten the length of the lines on other PRs as well. 
@tengqm or @sftim may be able to chime in here

Reply:
The overall convention is to wrap long lines where appropriate. Line wrappings in the markdown source won't affect the HTML output. Empty lines count because they are treated in Markdown as paragraph separators.

Line wrappings in Markdown source are preferred because:

- It helps reviewers to provide feedbacks for new text or text changes in the future. 
  After all, `git diff` works on a line-by-line comparison basis. Github web UI is not
  good at handling long lines either. It is hard to provided "suggestion" style feedbacks
  for reviewers.
- It helps downstream localization teams where the localization teams need to track
  each and every change in the English upstream. Suppose someone changes a few words
  in the existing English text, if we don't break the long Markdown lines, he/she who
  is translating the changes to a different language has to rework the whole paragraph.
- Wrapping long lines doesn't have to be done strictly on a 80-character line width basis.
  Contributors are encouraged to apply line wrappings when an English sentence ends,
  or after a punctuation character etc, wherever it helps speed up change tracking and
  review.
- There are exceptions to line wrappings. For example, we encourage keeping a Markdown
  link on a single line. This is because we want to detect dangling links in the source.

I hope this gives you an idea why we prefer manual line wrappings and sometimes we
appreciate cleanup PRs to fix paragraphs that are not properly wrapped. It is not a
matter of taste anyway.

Reply:
Thank you for the detailed explanation @tengqm. Very informative.